config/vufind/EDS.ini

```diff
@@ -187,17 +187,16 @@ top_rows = 2
 ; Should we OR together facets rather than ANDing them? Set to * for
 ; all facets, use a comma-separated list to apply to some of the facets, set
 ; to false or omit to disable ORed facets.
-; NOTE: In EDS, facet behavior is controlled through the EBSCO admin panel,
-; but you should configure this setting to match the EBSCO settings so that
-; VuFind displays match actual behavior.
 ;orFacets = *
 
 ; These settings affect the way facets are displayed on the advanced screen
 [Advanced_Facet_Settings]
 ; Should we OR together facets rather than ANDing them? Set to * for
 ; all facets, use a comma-separated list to apply to some of the facets, set
 ; to false or omit to disable ORed facets.
-orFacets = *
+; Note that ContentProvider and SourceType are forced to be OR facets, ignoring
+; this setting, due to special handling by the EDS API.
+; orFacets = *
```

Comment:
Is there a reason you changed the default advanced search behavior, or that you only put the "special handling" note in the advanced section but not in the regular section? I think it might be worth repeating, and I don't think we want to change the advanced facet behavior (unless there's a conscious reason why you did).

Reply:
This was mostly a mistake!  That said I haven't checked the advanced search behavior yet, will add a to-do.  I'm not sure why the prior default was different for basic vs. advanced, and I'm not sure if backwards compatibility matters when the point of the PR is to make the orFacets logic actually work for the first time -- BUT maybe it actually did work for advanced, who knows.  TBD

Reply:
In the Solr backend, advanced search uses OR facets so the user can multi-select values from the advanced form, which is a bit more flexible and useful than picking only one value in that particular interface. It's likely that when EDS was set up, those Solr defaults just got copied over without much thought, and nobody noticed that they didn't actually work as expected, because advanced search faceting is not a heavily used feature. If we can make things work correctly, it probably makes sense to continue to have parallel defaults between Solr and EDS.